File: python/46a_1702A_RoundDownThePrice.py
from math import pow, log10, floor

def main():
    ans = []
    t = int(input())
    for _ in range(t):
        m = int(input())
        ans.append(m - int(pow(10, floor(log10(m)))))
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()

File: cpp/46a_1702A_RoundDownThePrice.cpp
#include<bits/stdc++.h>

int main(){
    int t, m, k;
    std::vector<int> ans;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        std::cin>>m;
        k = log10(m);
        ans.push_back(m - pow(10, k));
    }
    for (auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}